packs/other-effects/effect-catch-your-breath.json

```diff
@@ -0,0 +1,41 @@
+{
+    "_id": "A81HpA9cBUgGRsqx",
+    "img": "icons/sundries/gaming/playing-cards-brown.webp",
+    "name": "Effect: Catch your Breath",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.journals.JournalEntry.BSp4LUSaOmUyjBko.JournalEntryPage.LafJ1PXfnR4Ogyzh]{Catch your Breath}</p>\n<p>The creature you choose must have fewer than half their total Hit Points. The creature gains a number of temporary Hit Points equal to twice your level, which last until the end of the combat.</p>"
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "encounter",
+            "value": -1
+        },
+        "level": {
+            "value": 1
+        },
+        "publication": {
+            "license": "OGL",
+            "remaster": false,
+            "title": "Pathfinder Hero Point Deck"
+        },
+        "rules": [
+            {
+                "key": "TempHP",
+                "value": "2*@actor.level"
```

Comment:
This might need to origin?

Reply:
It could, but we can't reliably say where the effect will be coming from anyway. Could come straight from the journal or a roll of the roll table, but neither of these would have origin data, right?